.github/workflows/rust.yml

```diff
@@ -25,10 +25,9 @@ jobs:
               AS_DEPENDENCY: true
               DO_NO_STD: true
               DO_DOCS: true
-          - rust: 1.29.0
+          - rust: 1.41.0
```

Comment:
I recommend using 1.41.1 since AFAIR 1.41.0 had miscompilation bugs.

Reply:
My bad, that is a mistake. Thanks for noticing.